src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -1044,6 +1068,179 @@ components:
           type: string
           description: A specific type of in-prison education/training that does not fit the given 'inPrisonEducation' types. Mandatory when 'inPrisonEducation' includes 'OTHER'.
 
+    UpdateCiagInductionRequest:
+      title: UpdateCiagInductionRequest
+      description: A request to update a CIAG Induction. Migrated from hmpps-ciag-careers-induction-api.
+      type: object
+      properties:
+        reference:
+          type: string
+          format: uuid
+          description: The Induction's unique reference. Currently optional (to avoid breaking the existing CIAG API), but will be mandatory in future (and will most likely become a path variable).
```

Comment:
As per the description here, I've made this optional for now. My plan is to overwrite every field in the DB, except this one if it's not provided in the request. That might not sound ideal, but it's hopefully only a temporary measure.